packages/apps/plugins/plugin-presenter/src/PresenterPlugin.tsx

```diff
@@ -22,6 +23,11 @@ type PresenterState = {
   presenting: boolean;
 };
 
+const isLocalMarkdownContent = (data: unknown): data is { content: EditorModel } =>
```

Comment:
Not sure, but I think EditorModel shouldn't appear in ECHO objects. cc @wittjosiah 

```suggestion
const isLocalMarkdownContent = (data: unknown): data is { content: TextObject } =>
```